src/Executables/Game/ParserUtils.cs

```diff
@@ -71,6 +74,73 @@ public static async Task<ImmutableArray<CommonDropEntry>> GetCommonDropsAsync(Te
 
         return list.ToImmutableArray();
     }
+    
+    public static async Task<ImmutableArray<SkillData>> GetSkillsAsync(string path, CancellationToken token = default)
+    {
+        var list = new List<SkillData>();
+        
+        if (!File.Exists(path))
+        {
+            return [..list];
```

Comment:
Just return `[]` - JIT will optimize this but but it looks unclean.

Reply:
I agree! Since this is coming from the ParserUtils class that only has static methods, we don't even have a way of logging something related to it, unless it `throws` (and we `catch` + log in the calling class) instead of returning an empty list ?

Reply:
You could make it a service with an interface which would totally be fine :)